### ISSUE TITLE
Your code runs successfully !But I want to know how long did your training code run?how many epochs did you set?

### ISSUE THREAD

**Issue by chengerr** [zhixuhao/unet#33]

**Comment by chengerr** [comment]
Your code's result did not perform well.  The loss function in your code seems not same as the 'unet ' paper.
